tyk-pro/templates/bootstrap-role.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: k8s-bootstrap-role
+  namespace: tyk
+rules:
+#
+# Give here only the privileges you need
+#
+- apiGroups: [""]
+  resources:
```

Comment:
Does it really need access to all of these resources and verbs?

Reply:
You're correct, not all of it was needed. Have narrowed the list to only the one that are really necessary.